bin/pint

```diff
@@ -150,15 +150,28 @@ if command_args['--region']:
 
 try:
     if command_args['images']:
-        data = ifsrequest.get_image_data(
-            framework,
-            image_state,
-            output_format,
-            region,
-            command_args['--filter'])
-        if '<images/>' in data or '"images": []' in data:
-            data = 'No information available. Please check your filter'
-        print(data)
+        no_info_regions = []
+        for reg in region.split(','):
+            data = ifsrequest.get_image_data(
+                framework,
+                image_state,
+                output_format,
+                reg,
+                command_args['--filter']
+            )
+            if '<images/>' in data or '"images": []' in data:
+                no_info_regions.append(reg)
+            else:
+                print(data)
+
+        if no_info_regions:
+            no_info_regions = (', ').join(no_info_regions)
+            data = "No information available for {} regions.".format(
+                no_info_regions
+            )
+            data += " Please check your filter"
+            print(data)
```

Comment:
If you choose multiple regions and one has no data while the others have data you end up with output in two different formats. An xml / json list followed by a string which makes the list not parsable.

Reply:
OK, I think that the message telling the user to check the filter may need to be removed,
and PINT could return data as it is, i.e.
```
{
  "images": []
}
```
if empty and JSON format

Thoughts ?

Reply:
IMHO if there's data we don't want an added warning message.

Reply:
This change also would/should affect [regions](https://github.com/SUSE-Enceladus/public-cloud-info-client/blob/master/bin/pint#L197) 
```python
if '<regions/>' in regions or '"regions": []' in regions:
            regions = ('No region information available. Images have '
                       'the same identifier in all regions')
```

Reply:
> IMHO if there's data we don't want an added warning message.

Agree but we should inform for a specific region there was no data, right ?
is the lack of response enough ?
let's say a user queries regions 'foo', 'bar' and 'foobar'
if 'bar' produces no data, we should return empty result, IMHO the lack of that info could lead to questions like
"did anything happen to the query ? to the response ?" an empty set says "query was OK but no result"

Reply:
I edited the code so now, for example, it produces

```bash
<?xml version='1.0' encoding='UTF-8'?>
<servers><server ip="IP_A" ipv6="" name="domain" region="region_A" type="smt"/>
  <server ip="IP_B" ipv6="" name="domain" region="region_A" type="smt"/>
  <server ip="IP_C" ipv6="" name="domain" region="region_A" type="smt"/>
<server ip="IP_D" ipv6="" name="domain" region="region_B" type="smt"/>
  <server ip="IP_E" ipv6="" name="domain" region="region_B" type="smt"/>
  <server ip="IP_F" ipv6="" name="domain" region="region_B" type="smt"/>
</servers>
```
or a message to change the filters only if there is not a single region that returns no empty information 
```bash
> bin/pint microsoft servers --filter='name~foo' --json --region=region_a,region_b
No information available. Please check your filter
> 
```